lib/JSHandle.js

```diff
@@ -312,9 +311,35 @@ class ElementHandle extends JSHandle {
    * @param {!Array<string>} filePaths
    */
   async uploadFile(...filePaths) {
-    const files = filePaths.map(filePath => path.resolve(filePath));
-    const objectId = this._remoteObject.objectId;
-    await this._client.send('DOM.setFileInputFiles', { objectId, files });
```

Comment:
This method hasn't been removed. I'm curious as to why we go down to JavaScript here.